pkg/collectors/virtualservices.go

```diff
@@ -0,0 +1,110 @@
+package collectors
+
+import (
+	"context"
+	"fmt"
+	"github.com/douglas-reid/istio-state-metrics/pkg/apis/networking/v1alpha3"
+	"github.com/douglas-reid/istio-state-metrics/pkg/client/clientset/versioned"
+	informers "github.com/douglas-reid/istio-state-metrics/pkg/client/informers/externalversions/networking/v1alpha3"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/client-go/tools/cache"
+)
+
+var (
+	descVirtualServiceHostsName          = "istio_pilot_virtual_service_hosts"
+	descVirtualServiceHostsHelp          = "Information about Hosts in Pilot VirtualServices"
+	descVirtualServiceHostsDefaultLabels = []string{"VirtualService", "hosts"}
+
+	descVirtualServiceHosts = prometheus.NewDesc(
+		descVirtualServiceHostsName,
+		descVirtualServiceHostsHelp,
+		descVirtualServiceHostsDefaultLabels,
+		nil,
+	)
+
+	descVirtualServicesInfo = prometheus.NewDesc(
+		"istio_pilot_virtualService_info",
```

Comment:
this should be `istio_pilot_virtual_service_info` to match other names.

Reply:
Thanks..Fixed